paasta_tools/cli/cmds/autoscale.py

```diff
@@ -72,9 +75,17 @@ def paasta_autoscale(args):
     else:
         log.debug(f"Setting desired instances to {args.set}.")
         body = {"desired_instances": int(args.set)}
-        res, http = api.autoscaler.update_autoscaler_count(
-            service=service, instance=args.instance, json_body=body
-        ).result()
+        try:
+            res, http = api.autoscaler.update_autoscaler_count(
+                service=service, instance=args.instance, json_body=body
+            ).result()
+        except HTTPNotFound:
+            paasta_print(
+                PaastaColors.red(
+                    f"ERROR: No such service or service is not configured to autoscale."
```

Comment:
Can you make this more user friendly?

How about


    Error: {instance} is not configured to autoscale, so 'paasta autoscale' cannot scale it up on demand.
    If you want to be able to boost this service, please configure autoscaling for it in the config file by setting min and max instances. Example:
    
         {instance}:
            min_instances: 5
            max_instances: 50